plugins/wallabag.koplugin/main.lua

```diff
@@ -188,12 +188,12 @@ function Wallabag:addToMainMenu(menu_items)
                     {
                         text_func = function()
                             local tags
-                            if not self.ignore_tags or self.ignore_tags == "" then
-                                tags = _("None")
+                            if not self.ignore_tags then
+                                tags = ""
                             else
                                 tags = self.ignore_tags
```

Comment:
In which case `return T(_("Ignore tags (%1)"), self.ignore_tags)` can probably just go here without extra variables.